src/atldld/config.py

```diff
@@ -0,0 +1,47 @@
+# The package atldld is a tool to download atlas data.
+#
+# Copyright (C) 2021 EPFL/Blue Brain Project
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+"""Configuration of the atldld CLI app and library."""
+import os
+import pathlib
+
+import appdirs
+
+
+def user_cache_dir(create: bool = True) -> pathlib.Path:
+    """Determine currently configured cache directory for atldld.
+
+    Parameters
+    ----------
+    create
+        If true the returned directory will be guaranteed to exist.
+
+    Returns
+    -------
+    pathlib.Path
+        The currently configured cache directory for atldld.
+    """
+    app_name = "atldld"
+    if "XDG_CACHE_HOME" in os.environ:
+        # appdirs reads XDG_CACHE_HOME only on Linux. Make it work for macOS too
```

Comment:
I don't think it should clash with other apps because we use the subdirectory `${XDG_CACHE_HOME}/atldld`, this should be the same with the normal cache folder in the sense that each app should write its own subfolder and not directly to the cache folder.

Regarding how commont it is, I dont' have a very complete overview either, but we do need some name for this, so taking one that's already standardized somehow was natural. Just a couple of points / google results
* `pip` uses `appdirs` and therefore respects `XDG_CACHE_HOME` on Linux.
* VSCode C++ Tools started using it: https://github.com/microsoft/vscode-cpptools/issues/3979
* [There is a spec for it](https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html)
* Just checked some of our Linux compute nodes - they all have some `XDG_` env variables set.
* [Anothony talked about it](https://www.youtube.com/watch?v=wT85Ynk-0MY) (you should be pretty much convinced now! 🤣 )

Reply:
Interesting:) Thanks for the references:) I am a simple man. If Anthony talks about it then I stop questioning it:)